core/src/main/scala/org/apache/spark/ui/jobs/JobProgressListener.scala

```diff
@@ -374,6 +374,10 @@ class JobProgressListener(conf: SparkConf) extends SparkListener with Logging {
         case kill: TaskKilled =>
           execSummary.reasonToNumKilled = execSummary.reasonToNumKilled.updated(
             kill.reason, execSummary.reasonToNumKilled.getOrElse(kill.reason, 0) + 1)
+        case TaskCommitDenied(jobID, partitionID, attemptNumber) =>
```

Comment:
give this a variable name like case commitDenied: TaskCommitDenied(jobID, partitionID, attemptNumber) that way you can use the variable commitDenied below in the toErrorString